docs/tutorials/fundamentals/installation.md

```diff
@@ -116,8 +116,16 @@ If you want to install napari with PySide2 as the backend you need to install it
 
 ```sh
 conda install -c conda-forge "napari=*=*pyside2"
-```  
+``` 
+````
 
+````{note}
+It can happend that `conda`'s build-in solver is not able to solve all denpendencies. This can happen even in a clean environment
```

Comment:
```suggestion
It can happend that `conda`'s built-in solver is not able to solve all denpendencies needed to install `napari`. This can happen even in a clean environment
```

Reply:
`s/happend/happen/`

Reply:
Fixed, thanks! 🙏

Reply:
Sorry for the typos >.<